### DISCUSSION TITLE
Using python to work with OWL/ontologies (basics)

### DISCUSSION THREAD

**Discussion by ghost** [jamesaoverton/obook#185] [Tutorial requests]
basic applications, overview of libraries

**Comment by lubianat** [comment]
by the way, the owlready2 python module has very good docs: https://owlready2.readthedocs.io/en/v0.37/intro.html

The paper is also really really good: https://www.researchgate.net/publication/319126534_Owlready_Ontology-oriented_programming_in_Python_with_automatic_classification_and_high_level_constructs_for_biomedical_ontologies
